test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Configuration.json

```diff
@@ -6,5 +6,6 @@
     "../../../../../artifacts/bin/AutoRest.CSharp/Debug/net7.0/Generator.Shared",
     "../../../../../artifacts/bin/AutoRest.CSharp/Debug/net7.0/Azure.Core.Shared"
   ],
-  "branded": false
+  "branded": false,
+  "azure-arm": false
```

Comment:
This looks strange, we don't want the name of Azure to exist in non-branding libraries.

Reply:
removed